lib/Plerd/Post.pm

```diff
@@ -664,153 +641,134 @@ sub send_webmentions {
 }
 
 sub add_webmention {
-    my $self = shift;
-    my ( $webmention ) = @_;
+    my ($self, $webmention) = @_;
 
-    $self->webmentions_by_source->{ $webmention->source } = $webmention;
+    $self->webmentions_by_source->{$webmention->source} = $webmention;
     $self->serialize_webmentions;
+
+    return;
 }
 
 sub update_webmention {
-    return add_webmention( @_ );
+    my ($self, $webmention) = @_;
+    return $self->add_webmention($webmention);
 }
 
 sub delete_webmention {
-    my $self = shift;
-    my ( $webmention ) = @_;
+    my ($self, $webmention) = @_;
 
-    delete $self->webmentions_by_source->{ $webmention->source };
+    delete $self->webmentions_by_source->{$webmention->source};
     $self->serialize_webmentions;
+    return;
 }
 
 sub serialize_webmentions {
     my $self = shift;
 
-    $self->_store( $WEBMENTIONS_STORE_FILENAME, $self->webmentions_by_source );
+    $self->_store($WEBMENTIONS_STORE_FILENAME, $self->webmentions_by_source);
+    return;
 }
 
 sub ordered_webmentions {
     my $self = shift;
 
-    return sort
-        {$a->time_published <=> $b->time_published }
-        values( %{ $self->webmentions_by_source } )
-    ;
+    my $ret = sort { $a->time_published <=> $b->time_published } values(%{$self->webmentions_by_source});
```

Comment:
This changes this method's behavior? It is supposed to return a list (the output of the call to sort()), not a scalar value.